### PR TITLE
Fix build with iodbc

### PR DESCRIPTION
**Pull request by proller**

### CODE REVIEW

File: driver/api/impl/info.cpp [hunk 1]
```diff
@@ -176,7 +176,10 @@ SQLRETURN SQL_API EXPORTED_FUNCTION_MAYBE_W(SQLGetInfo)(
             CASE_FALLTHROUGH(SQL_CONVERT_TIMESTAMP)
             CASE_FALLTHROUGH(SQL_CONVERT_INTERVAL_YEAR_MONTH)
             CASE_FALLTHROUGH(SQL_CONVERT_INTERVAL_DAY_TIME)
+
+#if defined(SQL_CONVERT_GUID)
             CASE_NUM(SQL_CONVERT_GUID, SQLUINTEGER, mask_common_CONVERT_dest)
```
Comment (traceon) [inline driver/api/impl/info.cpp]: That should probably switch places with some of the upper `CASE_FALLTHROUGH` for them to be terminated properly at `mask_common_CONVERT_dest`.

Comment (traceon) [inline driver/api/impl/info.cpp]: That should probably switch places with some of the upper CASE_FALLTHROUGH for them to be terminated properly at mask_common_CONVERT_dest.

**Comment by traceon** [comment]
I believe this one should fix this:
https://github.com/traceon/clickhouse-odbc/commit/741bbfb342d7dbcb7f9c2f459f52d5c9d1d14af3

**Comment by traceon** [comment]
> https://github.com/traceon/clickhouse-odbc/commit/741bbfb342d7dbcb7f9c2f459f52d5c9d1d14af3

That fix was cherry picked in https://github.com/ClickHouse/clickhouse-odbc/pull/225 .

**Comment by filimonov** [comment]
Closing for now. Let's check if the issue will be solved after #225

**Pull request closed without merging**